office365_connector.py

```diff
@@ -3263,6 +3263,46 @@ def _paginator(self, action_result, endpoint, limit=None, params=None, query=Non
 
         return phantom.APP_SUCCESS, list_items
 
+    def _handle_update_email(self, param):
+        self.save_progress(f"In action handler for: {self.get_action_identifier()}")
+        action_result = self.add_action_result(ActionResult(dict(param)))
```

Comment:
isn't param a dict already?

Reply:
```action_result = self.add_action_result(ActionResult(dict(param)))``` is a line that gets added by app wizard and it's the same in every action so I didn't change it. But yes, param i a dict